addons/redirect-mobile-forums/userscript.js

```diff
@@ -1,3 +1,5 @@
-export default async function ({ addon, console, msg }) {
-  window.location.replace(window.location.href.replace("m/", ""));
+export default async function ({ addon, console }) {
+  if (window.location.pathname.startsWith("/discuss/m/")) {
```

Comment:
```suggestion
  if (window.location.pathname.startsWith("/discuss/m/") || window.location.pathname === "/discuss/m") {
```

Reply:
`startsWith()` also works if it's the full string and since it replaces `m/` checking for a path without a trailing slash isn't very useful without updating the replace.

Reply:
Right